test/system/static.test.ts

```diff
@@ -633,3 +633,71 @@ test("static scanning NGINX with dockerfile analysis matches expected values", a
     "found gnupg1 in dockerfile packages",
   );
 });
+
+test("inspect registry.suse.com/suse/sle15:15.1", (t) => {
```

Comment:
I wonder if we could name the test with something around its goal, e.g. "able to scan SUSE images", the specific image chosen to prove this is already listed inside the test.